### ISSUE TITLE
Missing alt text for user profile

### ISSUE THREAD

**Issue by mgifford** [elmsln/elmsln#2465]
Bug:
- ELMSLN version/commit reported on admin/reports/status

Latest commit 0be9880  on Jan 26

- What happens

Logging in when you first set this up you get:

```
<div class="user-profile--photo circle">
      <img src="http://online.elmsln.local/sites/all/themes/elmsln_contrib/foundation_access/img/user.png" class="ferpa-protect circle">    </div>
```

- What you expected to happen

Probably it should be null

```
<div class="user-profile--photo circle">
      <img src="http://online.elmsln.local/sites/all/themes/elmsln_contrib/foundation_access/img/user.png" class="ferpa-protect circle" alt="">    </div>
```

or as a place holder `alt="default admin icon"`

Feature request:
- What you are trying to do / where it should go

Just loading up the page & looking at the accessibility of the admin interface. 

When is it getting upgraded to D8 btw?

**Comment by btopro** [comment]
thinking null since it's decorative. D8 isn't on my radar to not give a rant-y answer. highest priority for dev direction is to decouple D7 as much as possible, reduce complexity across the platform and then years down the road talk D8 or D9 if either still make sense (which I imagine they will). If they don't, backdrop will be reviewed at that time but we're real far from that

**Comment by mgifford** [comment]
Yup totally.  alt="" is totally legit. This was just a really quick review btw.  Might be a bigger issue when there are user photos there, but I haven't gotten that far.
